src/shared_modules/content_manager/src/actionOrchestrator.hpp

```diff
@@ -170,6 +168,12 @@ class ActionOrchestrator final
                     // LCOV_EXCL_STOP
             }
         }
+        catch (const offset_processing_exception& e)
+        {
+            logWarn(WM_CONTENTUPDATER, "Offset processing failed. Triggered a snapshot download.");
+            cleanContext();
```

Comment:
I simulated the failure in processing the offsets, which triggered an update through snapshots. During this process, I discovered that the snapshot update does not start due to an error when attempting to decompress the file. I suspect the issue is related to the temporary context that stores the downloaded files not being properly cleared.

![image](https://github.com/user-attachments/assets/1936da1a-2083-43f4-8450-180595b7b400)


I believe the temporary context should also be cleared at this point in the code.

I did the following test and it seems to work.

![image](https://github.com/user-attachments/assets/5bc1d3cc-0fcc-4573-acb9-96df3675ccc0)

Reply:
I will attempt to fix this! Thanks for the proposal